parser/parse_select.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) DataStax, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package parser
+
+import (
+	"errors"
+)
+
+// Determines is the proxy handles the select statement.
```

Comment:
Handled here means it's one of the system or peers tables?

Reply:
It's always tables in "system", but could be "local", "peers", as well as legacy schema tables. I've added a comment to clarify this.